nixos/modules/services/backup/btrbk.nix

```diff
@@ -84,6 +87,18 @@ let
     '';
   };
 
+  streamCompressMap = {
+    gzip = pkgs.gzip;
+    pigz = pkgs.pigz;
+    bzip2 = pkgs.bzip2;
+    pbzip2 = pkgs.pbzip2;
+    bzip3 = pkgs.bzip3;
+    xz = pkgs.xz;
+    lzo = pkgs.lzo;
+    lz4 = pkgs.lz4;
+    zstd = pkgs.zstd;
+  };
```

Comment:
just in case, I think adding an or or some kind of error if people use something that is not contained in the list would be nice

Reply:
this is probably good covered by the enum type of stream_compress? what do you think
```
       error: A definition for option `services.btrbk.instances.remote_myhost.settings.stream_compress' is not of type `one of "gzip", "pigz", "bzip2", "pbzip2", "bzip3", "xz", "lzo", "lz4", "zstd", "no"'. Definition values:
       - In `/home/onny/projects/nixpkgs/vm-btrbk.nix': "lz5"
 ```

Reply:
Nice, then we can ignore my comment.